src/Illuminate/Support/Facades/Cache.php

```diff
@@ -14,10 +14,10 @@
  * @method static \Illuminate\Cache\CacheManager extend(string $driver, \Closure $callback)
  * @method static bool has(array|string $key)
  * @method static bool missing(string $key)
- * @method static mixed get(array|string $key, mixed $default = null)
+ * @method static mixed get(array|string $key, mixed|\Closure $default = null)
  * @method static array many(array $keys)
  * @method static iterable getMultiple(iterable $keys, mixed $default = null)
- * @method static mixed pull(string $key, mixed $default = null)
+ * @method static mixed pull(array|string $key, mixed|\Closure $default = null)
```

Comment:
This is an example of a redundant type that we are intentionally leaving in place.